source/include/core_http_client.h

```diff
@@ -705,6 +817,24 @@ HTTPStatus_t HTTPClient_Send( const TransportInterface_t * pTransport,
  * - #HTTP_HEADER_NOT_FOUND (Header is not found in the passed response buffer.)
  * - #HTTP_INVALID_RESPONSE (Provided response is not a valid HTTP response for parsing.)
  * - #HTTP_PARSER_INTERNAL_ERROR(If an error in the response parser.)
+ *
+ * <b>Example</b>
+ * @code{c}
+ * HTTPStatus_t httpLibStatus = HTTP_SUCCESS;
```

Comment:
Our style guide states: `Names should avoid using abbreviations.`

However, this is sample code and probably does not need to reflect what the style guide states (We aren't using multi-line comments either). I leave it up to you.

Reply:
Good luck policing "cert"

Reply:
I have no idea why we make up rules that are meant to be broken